dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieClient.java

```diff
@@ -57,6 +57,8 @@
 
   private static final int LONG_POLLING_WAIT_TIME_IN_SECONDS = 30;
 
+  private static String revision = "0";
```

Comment:
IS_FIRST_PULL  revision  is better to use instance property, not static. When KieClient has only one instance, instance property is better. When KieClient has many instances(not possible), static is not good eitheir.